### PR TITLE
Server side render initial view

### LINKED ISSUE #7: setup deployment process for static site

### PR DESCRIPTION
**Pull request by wbprice**
Makes langa.io isomorphic, adds missing viewport tag to make it responsive.
Currently live in staging at https://langa-io-staging.herokuapp.com/
Fixes #7

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
+const gulp = require('gulp');
+const babel = require('gulp-babel');
+const webpack = require('webpack-stream');
+const imagemin = require('gulp-imagemin');
+const watch = require('gulp-watch');
+const batch = require('gulp-batch');
+
+const babelConfig = {
```
Comment (ryanwilliamquinn) [inline gulpfile.js]: do we need babel config here and in .babelrc?  seems like a pain to maintain.  It looks like it is slightly different than what we have in .babelrc too, is that intentional?

File: gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    .pipe(babel(babelConfig))
+    .pipe(gulp.dest('dist/components'));
+});
+
+gulp.task('moveStrings', () => {
+  return gulp.src('./src/assets/strings.js')
+    .pipe(gulp.dest('dist/assets/'));
+});
+
+gulp.task('moveStyles', () => {
```
Comment (ryanwilliamquinn) [inline gulpfile.js]: why don't these scss files need to be compiled? oh maybe they are compiled by webpack later? i guess i am confused about what 'compileStyles' task does. it doesn't look like it compiles anything?

**Comment by ryanwilliamquinn** [comment]
The staging site looks good. I'm confused about which plugin is actually doing sass preprocessing.

File: gulpfile.js (hunk 2 above)
Comment (wbprice) [reply gulpfile.js]: This can probably be removed.  `css-modules-transform` is compiling the SCSS now.

**Comment by ryanwilliamquinn** [comment]
Looks good to me now, thanks Blaine.  Good work.

**Comment by wbprice** [comment]
@ryanwilliamquinn I discovered an issue in staging involving the email form.  Maybe new, maybe old.  The form POSTs to `api.langa.io` and receives a 200 response. `{"email":"hello@langa.io","status":"queued","reject_reason":null}`
The sent indicator doesn't update.

**Pull request merged**